main.nf

```diff
@@ -1,13 +1,34 @@
-
 // Find your tower s3 bucket and upload your input files into it
 // The tower space is PHI safe
 nextflow.enable.dsl = 2
 
+// Synapse ID for Submission View
 params.view_id = "syn51356905"
-params.input_dir = "${projectDir}/input"
+// Synapse ID for Input Data folder
+params.input_id = "syn51390589"
+// CPUs to dedicate to RUN_DOCKER
 params.cpus = "4"
+// Memory to dedicate to RUN_DOCKER
 params.memory = "16.GB"
 
+
+// downloads synapse folder given Synapse ID and stages to /input
+process SYNAPSE_STAGE {
```

Comment:
In the future, do you envision us to use nf-synstage instead of this step?

Reply:
I was originally thinking that, but for this workflow, I think it might make more sense to leave it as coming from a Synapse project. The reasons I can think of for this are:

- Might make it easier for people running a challenge - you only have to worry about the Challenge's Synapse project and configuring the workflow to use the submission view and data from that project.
- This workflow only really needs to interact with Synapse since the challenge submissions come directly from Synapse.

So after thinking about it, I think using `nf-synstage` is probably not necessary. I think that the difference between this workflow and something like `nf-cwl-wrap` is that we aren't staging a large amount of input data for different workflow runs to go in parallel, we are staging one input dataset and then containers to be run, so having a data manifest to run this off of seems less necessary.